spec/models/game_spec.rb

```diff
@@ -43,4 +43,33 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      game = FactoryGirl.create(:game)
+      expect(game.pieces.count).to eq 32
+    end
+
+    it "Pawn is in correct position" do
+      game = FactoryGirl.create(:game)
+      expect(game.pieces.first.type).to eq("Pawn")
+
+      expected = [2, 1]
+      expect([game.pieces.first.rank, game.pieces.first.file]).to eq(expected)
+    end
+
+    it "Rook is in correct position" do
+      game = FactoryGirl.create(:game)
+      expect(game.pieces.last.type).to eq("Rook")
+
+      expected = [8, 8]
+      expect([game.pieces.last.rank, game.pieces.last.file]).to eq(expected)
```

Comment:
This is more of a style preference, but I think it's better to split out multiple comparisons into multiple `expect` statements. That way, they will get executed one at a time, and it will be easier to tell from the test results which one caused the test to fail.